src/modules/endpoint/sagas/watchFetchCities.js

```diff
@@ -14,11 +14,14 @@ import { baseUrl } from '../constants'
 
 function * fetchCities (dataContainer: DataContainer): Saga<void> {
   try {
-    const payload: Payload<Array<CityModel>> = yield call(() => request(createCitiesEndpoint(baseUrl)))
-
-    const cities: Array<CityModel> = payload.data
-
-    yield call(dataContainer.setCities, cities)
+    let cities: Array<CityModel>
+    if (yield call(() => dataContainer.citiesAvailable())) {
+      cities = yield call(() => dataContainer.getCities())
```

Comment:
The user should be able to force a refresh (using pull-to-refresh) of the cities.
If you want, you can implement that in this task or in another task.
If you choose the latter please create one for that :)